CONTRIBUTING.md

```diff
@@ -32,19 +22,57 @@ Content in this collection that is stable and meets other acceptance criteria ha
 to be promoted and migrated into `amazon.aws`.
 
 ## Submitting Issues
-All software has bugs, and the `amazon.aws` collection is no exception. When you find a bug, 
+All software has bugs, and the `amazon.aws` collection is no exception. When you find a bug,
 you can help tremendously by [telling us about it](https://github.com/ansible-collections/amazon.aws/issues/new/choose).
 
-If you should discover that the bug you're trying to file already exists in an issue, 
-you can help by verifying the behavior of the reported bug with a comment in that 
+If you should discover that the bug you're trying to file already exists in an issue,
+you can help by verifying the behavior of the reported bug with a comment in that
 issue, or by reporting any additional information
 
-## Pull Requests
+## Writing New Code
+
+New modules should be submitted to the [community.aws](https://github.com/ansible-collections/community.aws) collection.
+
+For new features and bug fixes on existing modules,
+clone this repository and try to run unit tests and integration tests by following
+[these instructions](https://docs.ansible.com/ansible/latest/community/create_pr_quick_start.html).
+When you get to this part:
+
+```
+ansible-test integration name_of_test_subdirectory --docker -v
+```
+
+Run this from the `tests` directory of this repository.
+Substitute `name_of_test_subdirectory` for the name of the relevant directory within `tests/integration/targets`.
+You'll get this error:
+
+```
+WARNING: Excluding tests marked "cloud/aws" which require config
+(see "/home/dev/ansible/ansible/test/lib/ansible_test/config/cloud-config-aws.ini.template"): ec2_group
+```
+This is because the unit tests don't automatically detect the AWS credentials on your machine
+(unlike plain `boto3` and the `aws` cli).
+You need to explicitly create credentials and load them in.
+To do this, copy the file mentioned in that error message,
+into the clone of this repo, under `tests/integration/cloud-config-aws.ini`.
+Modify the `@` variables, pasting in an IAM secret credential.
+If you don't need the `secret_token` (most IAM users don't), comment that line out.
+
+You can use an AWS account that already has unrelated resources in it.
+The tests should not touch pre-existing resources, and should tidy up after themselves.
+(Of course for security reasons you may want to run in a dedicated AWS account.)
+
+If you're only writing a pull request for one AWS service
+you are able to create credentials only with permissions required for that test.
+For example, to test the Lambda modules, you only need Lambda permissions,
+and permissions to create IAM roles.
```

Comment:
It's also possible to use the same permissions that are used by CI - 

https://github.com/mattclay/aws-terminator/blob/master/hacking/aws_config/setup-iam.yml

Reply:
That's a link to a playbook, not the policies. I'm assuming [these](https://github.com/mattclay/aws-terminator/tree/master/aws/policy) are the policies. I've added a link to those.